src/sonic-py-common/sonic_py_common/device_info.py

```diff
@@ -479,15 +479,15 @@ def get_platform_info(config_db=None):
     if hw_info_dict:
         return hw_info_dict
 
-    from .multi_asic import get_num_asics
+    from .multi_asic import get_asic_presence_list
 
     version_info = get_sonic_version_info()
 
     hw_info_dict['platform'] = get_platform()
     hw_info_dict['hwsku'] = get_hwsku()
     if version_info:
         hw_info_dict['asic_type'] = version_info.get('asic_type')
-    hw_info_dict['asic_count'] = get_num_asics()
+    hw_info_dict['asic_count'] = len(get_asic_presence_list())
```

Comment:
can we put this change in try/catch to ensure `show version` when chassis_db is not up?

Reply:
@arlakshm thanks. have updated accordingly.